hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -240,8 +260,9 @@ data ErrCoverFee
   = ErrNotEnoughFunds ChangeError
   | ErrNoFuelUTxOFound
   | ErrUnknownInput {input :: TxIn}
-  | ErrScriptExecutionFailed {scriptFailure :: (PlutusPurpose AsIx LedgerEra, TransactionScriptFailure LedgerEra)}
+  | ErrScriptExecutionFailed {redeemerPointer :: Text, scriptFailure :: Text}
```

Comment:
I'd like to use a different too, but unreasonably hard IMO. It would make the whole type parametric by `era` or we could do an existential `redeemerPointer :: forall a. Show a => a` (or something in between).

Ultimately, we use this data constructor only to fill another constructor with `Text`. So we did not lose anything by converting to `Text` early (right now).